### PR TITLE
OSD-Migration: Migrating Network Multicast to Main

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR will merge the network files from OSD into MAIN.

PREVIEW:

### CODE REVIEW

File: modules/nw-enabling-multicast.adoc [hunk 1]
```diff
@@ -17,10 +17,16 @@ endif::[]
 
 You can enable multicast between pods for your project.
 
+ifdef::openshift-origin,openshift-online,openshift-enterprise,partner-roks,openshift-webscale[]
```
Comment (kalexand-rh) [inline modules/nw-enabling-multicast.adoc:20]: Consider simplifying this conditional. I think should accomplish the same thing with less duplication:

```
.Prerequisites
* Install the OpenShift CLI (`oc`).
* You must log in to the cluster with a user that has the `cluster-admin`
ifdef::openshift-rosa,openshift-dedicated[]
or the `dedicated-admin`
endif::[]
role.
```

**Review by skrthomas — COMMENTED** [review]
I'm not really an ifdef wiz, but Kathryn's suggestion seems reasonable. The previews you sent/changes in the file LGTM.

**Comment by pneedle-rh** [comment]
This looks great, thank you Eric! Please do not merge this PR until after the `dedicated-4` freeze. We can check if any further updates are made to the corresponding files in `dedicated-4` after the freeze. If so, we might need to add that content in further conditionals in this PR before we merge it. Thanks!

**Comment by pneedle-rh** [comment]
This PR relates to https://issues.redhat.com/browse/OSDOCS-2869.

**Pull request merged**